helix-core/src/main/java/org/apache/helix/BaseDataAccessor.java

```diff
@@ -166,6 +167,7 @@
    * @param options Set the type of ZNode see the valid values in {@link AccessOption}
    * @return A list of children of the parent ZNode
    */
+  @Deprecated
   List<T> getChildren(String parentPath, List<Stat> stats, int options);
 
   /**
```

Comment:
What is the unit of retryInterval, second or millisecond?  Should be use long?  Usually have three consecutive parameters with the same type is not easy to use.

Reply:
Lei, let's address the interface design question in separate PR if necessary? This design is not changed here.

In the current change, I am retrying once in the callers. But zero retry also works, I think, since we have added the async internal retry on connectivity issues. Do you think we shall just make it zero retry?

Reply:
Since we already checked in the zkclient async call retry PR, let me change to zero retry for the newly changed callers in this PR.